# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/media/test/crashtests/1860840.mp4
# Commit: fd44d547e093
# Full Hash: fd44d547e0931ddcb25e9dde2691ad9e99ec2e05
# Author: alwu <alwu@mozilla.com>
# Date: 2023-10-26 09:13:45
# Description:
#   Bug 1860840 - return error for invalid hvcc config when converting sample. r=media-playback-reviewers,padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D191794
# ==============================================================================

diff -r 691148c77012 -r fd44d547e093 dom/media/test/crashtests/1860840.mp4
Binary file dom/media/test/crashtests/1860840.mp4 has changed